tensorflow/python/ops/math_ops.py

```diff
@@ -3212,6 +3212,12 @@ def sigmoid(x, name=None):
 
   Returns:
     A Tensor with the same type as `x`.
+  
+  Usage Example:
+    ```python
+    x = tf.constant([125.92, -76.32, 44.12, -87.3, -82.72, 0.0, 112.23], dtype=tf.float32)
```

Comment:
And also add `>>>` to start of the command lines so it will run doctests on them.